resources/lang/vi/validation.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'attributes' => [
+        'address'                  => 'địa chỉ',
```

Comment:
```suggestion
        'address' => 'địa chỉ',
```
sửa tương tự các line khác nha.